### DISCUSSION TITLE
「直播弹幕发送栏」可以删除（原：「直播弹幕发送栏」和「直播全屏包裹」可以删除）

### DISCUSSION THREAD

**Discussion by timongh** [the1812/Bilibili-Evolved#4060] [综合讨论 / General]
B站不知道什么时候已经自带了这两个功能，这两个组件不再需要了。

**Comment by the1812** [comment]
直播弹幕发送栏是针对网页全屏, b 站只自带了全屏的
直播全屏包裹打开的是包裹, b 站自带的是送礼界面

**Comment by timongh** [comment]
**EDIT**: 我没注意，B 站自带的那个确实不是包裹。不过网页全屏的弹幕发送栏到是一直都有。

![image](https://user-images.githubusercontent.com/46739861/225495662-7ae56b7d-afcf-4072-a76c-d9bf8e5fab5e.png)
![image](https://user-images.githubusercontent.com/46739861/225495892-4b39efae-824d-4806-914a-d0c0f6354b3f.png)

不知道 B 站什么时候改的，总之现在都有。这两个截图中我都没开脚本。

**Reply by the1812** [reply]
直播弹幕发送栏就是在网页全屏下不想打开侧栏时使用的
